docs/proposals/updating_upstream_images.md

```diff
@@ -0,0 +1,54 @@
+Updating upstream image tags for supported branches
+---------------------------------------------------
+Our logging team received a few issue and bug reports for a stable upstream 
+branch, where the proper code was already merged in github, but propagating
+the code to upstream images was out of sync with openshift ansible releases.
+
+Upstream image tags
+-------------------
+For upstream images, we have the `latest` image tag and then image tag based
+on the `git tag` (for example `v3.6.1`). The `latest` is frequently updated
+when we merge a PR to the `master` branch, the `v3.6.1` tag is created once and 
+does not get updated. For downstream images, we also have updating tags based
+on `git branch` (for example branch `rhaos-3.6-rhel-7` to image tag `v3.6`) 
+and these get updated by our release engineers.
+
+I would like to propose similar updating tags for the upstream images based on 
+the `git branch`.
+
+Proposal
+--------
+We would like to keep the original workflow for `master` and `git branch` for
+upstream image tags and add a separate workflow for when we merge a PR to
+any release branch.
+
+Mapping from git to the image tag:
+```
+git                   | image tag
+----------------------|--------------------
+branch - master       | updateable - latest
+branch - release-X.Y  | updateable - vX.Y         <--- proposed new image tag
```

Comment:
So the thought here is that `vX.Y` is a floating tag -- it can point to either the latest tagged release of `vX.Y.Z` or the last merge from `release-X.Y` ?

Reply:
yes, that's my understanding - that's the way downstream images work as well, so it would be consistent

Reply:
exactly, that is what I was trying to describe. Do you have any suggestions how to improve the proposal and make the wording clearer so it is easier to understand?

Reply:
Consider explicitly noting this is a floating tag

Reply:
I tried to explicitly state that with "updateable", will swap that for "floating"